src/common/successAlert.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+
+export interface SuccessAlertProps {
+    message: string;
+    show: boolean;
+}
+
+export default function SuccessAlert(props: SuccessAlertProps) {
```

Comment:
Again, `react-toastify` is a WAY more quicker solution
Reinventing the wheel here